src/Components/Admin/navbar.js

```diff
@@ -61,7 +61,7 @@ export default function Navbar({ home }) {
         }
     }
 
-    const { isAdmin } = useContext(UserContext);
+    const { user } = useContext(UserContext);
```

Comment:
Can you explain why we changed this yet again? I remember we changed it to `isAdmin` in the previous auth PR

Reply:
We did `isAdmin` since `user` was set (as a deprecated field that was no longer being used). Now that we removed the user document generation and the user object, I think user is a more logical name for this field.